### PR TITLE
feat: Solution based Cdot dashboard

### PR DESCRIPTION
**Pull request by Hardikl**
Incorporated the panels outlined here: https://grafana.com/grafana/dashboards/10181-netapp-overview/

image

image

### CODE REVIEW

**Comment by cgrinds** [comment]
minor: cDOT instead of Cdot to be consistent with other cDOT dashboards

**Comment by Hardikl** [comment]
> minor: cDOT instead of Cdot to be consistent with other cDOT dashboards

Done

**Comment by Hardikl** [comment]
image

image

image

image

image

**Comment by Hardikl** [comment]
> image image
> 
> 1. This should add information about the object also. Node IOPs?
> 2. Tables are missing topk range based on hiddden variable.

Removed the 2 tables and added SVM level and Volume level avgLatency, throughput, Iops.

Also, Organized as Cluster Metrics(cluster scoped), SVM Metrics(SVM scoped), Volume Metrics(Volume scope) and Capacity Metrics(Used % for Volume, Aggr, SVM)[This is Current value].

**Comment by Hardikl** [comment]
Added the capacity graph panels 
image

For removing the green color from table:
- Grafana supports to add color to whole column(multi color) or without color, but it's not able to add color just few rows.

File: conf/zapiperf/cdot/9.8.0/cifs_node.yaml [hunk 1]
```diff
@@ -6,7 +6,7 @@ object:                   node_cifs
 counters:
   - instance_name            => node
   - cifs_op_count            => op_count
-  - cifs_ops                 => ops
+  - cifs_ops                 => total_ops   # "cifs_ops" already used in system_node.yaml
```
[the remark this answers is not in the document]
Comment (Hardikl) [reply conf/zapiperf/cdot/9.8.0/cifs_node.yaml:9]: done

**Review by cgrinds — APPROVED** [review]

**Pull request merged**